README.md

```diff
@@ -63,10 +63,10 @@ docker start redis-florist-server
 
 ### Start back-end and front-end servers
 
-Use Yarn to run both the back-end and front-end on server mode:
+Use Yarn to run both the back-end and front-end on production server mode:
 
 ```shell
-yarn dev
+yarn prod
```

Comment:
Feel free to ignore if you think its overkill, but maybe add a header in the contributions file like [FL4Health](https://github.com/VectorInstitute/FL4Health#contributing) that points people to the CONTRIBUTING.md file for setting up the dev environment and running the app in dev mode

Reply:
Another related documentation ask (that you can ignore if you thinks its overkill), is to add that a UI exists through FastAPI where you can make requests at localhost:PORT/docs once you start a client and/or server. Since the endpoint to start training has a lot of parameters, its alot nicer than hacking a long url together although some may not be aware

Reply:
Whatever works best! If it's in the one you are going to put up that works too

Reply:
Just added it here :)